__tests__/test.js

```diff
@@ -169,6 +169,23 @@ describe("/api/articles", () => {
         expect(articles).toBeSortedBy("created_at", { descending: true });
       });
   });
+  test('GET request with topic=mitch query should respond with an array of objects where the topic is mitch', () => {
+    return request(app)
+    .get('/api/articles?topic=mitch')
+    .then(({ body }) => {
+        const { articles } = body
+        articles.forEach((article) => {
+            expect(article.topic).toBe('mitch')
+        })
+    })
+  })
+  test('GET request with topic=notatopic query should respond with 404 and a msg of topic not found', () => {
+    return request(app)
+    .get('/api/articles?topic=notatopic')
+    .then(({ body }) => {
+        expect(body.msg).toBe('topic not found')
+    })
+  })
```

Comment:
I'd also expect to see a test for a valid topic that has no articles. What status code should that be? There is a topic of paper that you can test this with.